db/db.go

```diff
@@ -51,7 +51,6 @@ var (
 type DB struct {
 	io.Closer
 
-	net       app.Net
```

Comment:
This was actually not being set in latest master. We don't need it anyway since `connector.Net` is now accessible.